router/auth.js

```diff
@@ -5,17 +5,19 @@ const router = express.Router()
 
 router.post('/register',async(req,res)=>{
 
-   const{name,email,phoneno,password}=req.body;
-   if(!name||!email||!phoneno||!password){
+   const{name,email,createpassword,confirmpassword}=req.body;
+   if(!name||!email||!createpassword||!confirmpassword){
       res.status(601).json({error:"fill all the forms correctly"}) 
```

Comment:
A return statement should be placed here to avoid running the remaining code. And there is not status code 601. Instead, we should return status 400 that indicates a bad request from client side.
```suggestion
      return res.status(400).json({error:"fill all the forms correctly"}) 
```